Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM debian:bullseye
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    python-is-python3 \
-    python3 \
-    python3-pip \
-    && rm -rf /var/lib/apt/lists/*
+FROM python:3.9
+
 COPY . /ml-compiler-opt
-RUN python3 -m pip install -r /ml-compiler-opt/requirements.txt
+RUN pip install --no-cache-dir -r /ml-compiler-opt/requirements.txt
```

Comment:
What's the rationale here for using `--no-cache-dir`?

Reply:
The cache is used to store the installation files(`.whl`, etc.) of the modules that you install through pip and the source files (`.tar.gz`, etc.) neither of which are needed inside the container, so this helps to reduce the size of the layer.

Reply:
Makes perfect sense. Was thinking of the wrong file when I saw the line.